package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "cicero-word-add-in",
+  "version": "0.0.1",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/accordproject/cicero-word-add-in.git"
+  },
+  "license": "MIT",
```

Comment:
I think this should be under Apache, @dselman or @jeromesimeon could you confirm?

Reply:
yes, all the Accord Project contributions should be under Apache 2.0 license